test/scripts/generate_script_tests.py

```
@@ -230,172 +209,142 @@
+    "KTH_OP_OVER": "kth::error::insufficient_main_stack",
+    "KTH_OP_PICK": "kth::error::insufficient_main_stack",
+    "KTH_OP_ROLL": "kth::error::insufficient_main_stack",
+    "KTH_OP_ROT": "kth::error::insufficient_main_stack",
+    "KTH_OP_SWAP": "kth::error::insufficient_main_stack",
+    "KTH_OP_TUCK": "kth::error::insufficient_main_stack",
+    "KTH_OP_DUP2": "kth::error::insufficient_main_stack",
+    "KTH_OP_DUP3": "kth::error::insufficient_main_stack",
+    "KTH_OP_OVER2": "kth::error::insufficient_main_stack",
+    "KTH_OP_ROT2": "kth::error::insufficient_main_stack",
+    "KTH_OP_SWAP2": "kth::error::insufficient_main_stack",
+    "KTH_OP_CAT": "kth::error::insufficient_main_stack",
```

Ruff (0.15.4)

[error] Dictionary key literal `"KTH_OP_CAT"` repeated

Remove repeated key literal `"KTH_OP_CAT"`

(F601)